src/api/request-hooks/request-logger.js

```diff
@@ -38,7 +38,7 @@ class RequestLoggerImplementation extends RequestHook {
     }
 
     async onRequest (event) {
-        const userAgent = parseUserAgent(event._requestInfo.userAgent).toString();
+        const userAgent = parseUserAgent(event._requestInfo.userAgent).userAgent;
```

Comment:
Let's rename the `userAgent` variable to make the code more clear.
```js
const shortUserAgent = ....
```

Reply:
I removed this redundant variable:
```ts
{
    ...
    userAgent: parseUserAgent(event._requestInfo.userAgent).userAgent,
    ...
}
```

Reply:
Ok